pages/linux/truncate.md

```diff
@@ -0,0 +1,15 @@
+# truncate
+
+> Shrink or extend the size of a file to the specified size.
+
+- Set the size of an existing file, or create a new file with the specified size:
+
+`truncate -s 10G {{filename}}`
+
+- Extend the file size, fill with holes (which reads as zero bytes):
+
+`truncate -s +50M {{filename}}`
+
+- Shrink the file and discard the extra data:
```

Comment:
I am guessing it removes data from the end ? We should clarify that.

Reply:
Hrm. How to word this?  Maybe `Shrink the file by 2 GiB, starting from the end of the file. Will delete data when shrinking:`?

Reply:
`Shrink the file by 2GiB, by removing data from the end of file:` ?

Reply:
Yeah!